simulation/simple_sensor_simulator/include/simple_sensor_simulator/vehicle_simulation/vehicle_model/sim_model_delay_steer_map_acc_geared.hpp

```diff
@@ -0,0 +1,338 @@
+// Copyright 2023 The Autoware Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SIMPLE_PLANNING_SIMULATOR__VEHICLE_MODEL__SIM_MODEL_DELAY_STEER_MAP_ACC_GEARED_HPP_
+#define SIMPLE_PLANNING_SIMULATOR__VEHICLE_MODEL__SIM_MODEL_DELAY_STEER_MAP_ACC_GEARED_HPP_
+
+//#include <common/types.hpp>
+#include <deque>
+#include <fstream>
+#include <iostream>
+#include <queue>
+#include <simple_sensor_simulator/vehicle_simulation/vehicle_model/sim_model_interface.hpp>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include "eigen3/Eigen/Core"
+#include "eigen3/Eigen/LU"
+
+// copied from https://github.com/tier4/autoware.universe/blob/v0.17.0/common/interpolation/include/interpolation/interpolation_utils.hpp
```

Comment:
Please use doxygen [note command.](https://www.doxygen.nl/manual/commands.html#cmdnote)

Reply:
fix in https://github.com/tier4/scenario_simulator_v2/pull/1144/commits/8f992dc0fd474ac599a9470102e5abc78d42310f